Resources/public/templates/tabs/locations.hbt

```diff
@@ -73,3 +73,14 @@
         </div>
     </div>
 </div>
+
+<div class="ez-locations-box">
```

Comment:
No problem for me, but note i just followed what i saw in https://github.com/ezsystems/PlatformUIBundle/blob/master/Resources/public/templates/tabs/details.hbt#L54-L68 :)

Thought that was the convention :)